src/openstudio_lib/FacilityShadingGridView.cpp

```diff
@@ -498,6 +507,22 @@ void FacilityShadingGridController::addColumns(const QString& category, std::vec
                           boost::optional<std::function<void(model::ShadingSurface*)>>(
                             std::function<void(model::ShadingSurface*)>([](model::ShadingSurface* t_ss) { t_ss->remove(); })),
                           boost::optional<std::function<bool(model::ShadingSurface*)>>(), DataSource(allShadingSurfaces, true));
+      } else if (field == DISPLAYNAME) {
+        addLoadNameColumn(Heading(QString(DISPLAYNAME), false, false),                                        // heading
```

Comment:
Maybe just `addValueEditColumn` would work here?  Or `addNameLineEditColumn`?  To show the value without the icon?

Reply:
Yeah probably. I'll look into it